### PR TITLE
feat: add fortune teller lab

### LINKED ISSUE #419: chore: add random number generator lab to frontend cert(JS only)
Creating random numbers comes up a lot in the current beta JS curriculum and campers are still struggling with it.
So we will have a workshop teaching the Math object and we will have a lab all about random numbers.
There will also be lectures too and quiz questions to make sure they really understand the material this time around. 

Here is the intro to numbers and Math object JS workshop:
https://github.com/freeCodeCamp/CurriculumExpansion/pull/418

We need a random number generator lab that will immediately follow that.

Here is what the campers will know so far:

- let
- const
- assignment
- reassignment
- basic string usage
- console.log
- camelCase 
- string concatenation using the + operator
- template literals
- str.length property
- bracket notation and retrieving characters from strings
- str.indexOf method
- if/else
- Number()
- Math.floor
- Math.random
- getting a random number from a range
- prompt()
- ===

## Acceptance criteria

- [ ] create a new folder of the project in the `frontend-cert/js-projects` directory
- [ ] add your completed project in an `script.js` file in that new folder
- [ ] create a markdown file for your proposed user stories. You should call it `user-stories.md` and place it in your new project folder

### PR DESCRIPTION
**Pull request by zairahira**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #419

### CODE REVIEW

**Review by Dario-DC — COMMENTED** [review]
Just wondering why the use of `var`. I think we should use only `let` and `const`.

File: frontend-cert/js-projects/fortune-teller-app/user-stories.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+1. You should have a set of 5 fortunes as strings and assign them to different variables.
```
Comment (moT01) [inline frontend-cert/js-projects/fortune-teller-app/user-stories.md:1]:
```suggestion
1. You should declare five variables, each with a string value that describes a fortune.
```

set is a built-in object in JS, I think we should be a little more specific here.

File: frontend-cert/js-projects/fortune-teller-app/user-stories.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+1. You should select a random number between 
+1 and 5 inclusive and assign it to the variable `randomNumber`.
```
Comment (moT01) [inline frontend-cert/js-projects/fortune-teller-app/user-stories.md:3]:
```suggestion
1. You should select a random number between 1 and 5, inclusive, and assign it to the variable `randomNumber`.
```

File: frontend-cert/js-projects/fortune-teller-app/user-stories.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+You should use an `if/else if` statements to print a different fortune based on the value of `randomNumber`:
+   - If `randomNumber` is 1, print the first fortune.
+   - If `randomNumber` is 2, print the second fortune.
+   - If `randomNumber` is 3, print the third fortune.
+   - If `randomNumber` is 4, print the fourth fortune.
+   - If `randomNumber` is 5, print the fifth fortune.
+1. Log the output to the console.
```
Comment (moT01) [inline frontend-cert/js-projects/fortune-teller-app/user-stories.md:10]:
```suggestion
1. You should log the fortune to the console.
```

**Review by moT01 — COMMENTED** [review]
This is looking good @zairahira 👍 I left a few suggestions.

**Review by moT01 — APPROVED** [review]
🎉

**Pull request merged**